package.json

```diff
@@ -4,18 +4,18 @@
   "private": true,
   "cracoConfig": "./craco.config.js",
   "dependencies": {
-    "@swagger-api/swagger-editor": "^5.0.0-alpha.44",
+    "@swagger-api/swagger-editor": "^5.0.0-alpha.45",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "node-polyfill-webpack-plugin": "^2.0.1",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
     "react-scripts": "5.0.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "craco start --config ./craco.config.js",
```

Comment:
No need, config is already designated by https://github.com/vitjaz/swagger-editor-next-react-example/pull/1/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R5